deepbox/src/test/java/ch/cyberduck/core/deepbox/DeepboxListServiceTest.java

```diff
@@ -117,6 +137,27 @@ public void testListBoxes() throws Exception {
         }
     }
 
+    @Test
+    public void testListSharedBoxes() throws Exception {
+        final DeepboxIdProvider nodeid = new DeepboxIdProvider(session);
+        final Path shared = new Path(String.format("/ORG 1 - DeepBox Desktop App/%s", DeepboxListService.SHARED), EnumSet.of(Path.Type.directory, Path.Type.volume));
+        final AttributedList<Path> list = new DeepboxListService(session, nodeid).list(shared, new DisabledListProgressListener());
+        assertNotSame(AttributedList.emptyList(), list);
+        assertFalse(list.isEmpty());
+        assertNotNull(list.find(new SimplePathPredicate(new Path(String.format("/ORG 1 - DeepBox Desktop App/%s/Testing (1 Christian Gruber)", DeepboxListService.SHARED), EnumSet.of(Path.Type.directory, Path.Type.volume)))));
+        assertEquals(1, list.size());
+        for(final Path f : list) {
+            assertSame(shared, f.getParent());
+            assertFalse(f.getName().contains(String.valueOf(Path.DELIMITER)));
+            // no modification/creation date for Boxes
+            assertTrue(f.attributes().getModificationDate() < 0);
+            assertTrue(f.attributes().getCreationDate() < 0);
+            assertNotNull(nodeid.getFileId(new Path(f).withAttributes(new PathAttributes())));
+            assertEquals(f.attributes(), new DeepboxAttributesFinderFeature(session, nodeid).find(new Path(f.getAbsolute(), f.getType())));
+        }
+        session.close();
```

Comment:
Already in `@After` of superclass.

Reply:
In [212d0c7](https://github.com/iterate-ch/cyberduck/pull/16420/commits/212d0c7693cfd27a369d4a980059b27e2da79ab3).